recipes/oommf/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+TCLTKVERSION=8.5
+CONDA_ROOT_PATH=$(conda info --root)
+
+# Compile OOMMF
+export OOMMF_TCL_CONFIG=${PREFIX}/lib/tclConfig.sh
+export OOMMF_TK_CONFIG=${PREFIX}/lib/tkConfig.sh
+cd oommf
+tclsh${TCLTKVERSION} oommf.tcl pimake distclean
+tclsh${TCLTKVERSION} oommf.tcl pimake upgrade
+tclsh${TCLTKVERSION} oommf.tcl pimake
+tclsh${TCLTKVERSION} oommf.tcl +platform
```

Comment:
The `tclsh${TCLTKVERSION} oommf.tcl +platform` is a little bit like a test: it spits out diagnostic information about this install. (Another command we could run, and which is listed in the `test` section in `meta.yaml', is `tclsh${TCLTKVERSION} oommf.tcl +version`, which reports just the version number, but is also a test to check some basics of the installation.

There is no automatic test suit for OOMMF.